### DISCUSSION TITLE
Sonoff mini vs mini R2

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#1286] [Hardware & Firmware]
Salve,
sappiamo che una delle differenze principali fra il mini e il mini R2 è che R2 ha antenna interna mentre R1 aveva il filino che potevi portare fuori.
Qualcuno di voi ha fatto dei test per capire quale dei due ha una ricezione migliore?
Io ho testato da poco un R1 abbastanza lontano dall'access point (fuori casa in giardino) e prendo il segnale regolarmente.
Ho un po' paura a prendere degli R2 con antenna integrata, ho paura che la ricezione sia inferiore.

**Comment by KC-inDomus** [comment]
la è senza dubbio. ma se la distanza non è abissale, per me la questione è un finto problema.
